src/pretix/presale/templates/pretixpresale/event/checkout_questions.html

```diff
@@ -41,6 +41,11 @@ <h4 class="panel-title">
                     </div>
                 </div>
             {% endif %}
+            <div id="cp_copy" class="panel panel-default">
+                <div class="panel-body">
+                    <input type="checkbox" name="copy" id="copy_answers">  Copy Answers
```

Comment:
We need to have this translatable, you can use something like ``{% trans "Copy answers" %}`` for this.